### PR TITLE
ports/mimxrt/boards: Add mimxrt1060-evk board.

### PR DESCRIPTION
**Pull request by Albort12138**
- Enabled mimxrt1060-evk board.

### CODE REVIEW

File: ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,7 @@
+MCU_SERIES = MIMXRT1062
+MCU_VARIANT = MIMXRT1062DVJ6A
+
+JLINK_PATH = /media/RT1060-EVK/
```
Comment (dpgeorge) [inline ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.mk:4]: Perhaps this should use `?=` so it can be overridden?

**Comment by dpgeorge** [comment]
Thanks, it looks good, I assume you did basic tests?

**Comment by yanyao12138** [comment]
> Thanks, it looks good, I assume you did basic tests?

Yes, I have done the basic tests? I will update this PR based on your comments

File: ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.mk (hunk 1 above)
Comment (yanyao12138) [reply ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.mk]: Thanks for your comments, I have updated the PR.

**Comment by dpgeorge** [comment]
Rebased and merged in b3bc9808f2d1396a946ae899b7d81d043eac1ddd

**Pull request closed without merging**